### PR TITLE
Add recipe for modsecurity

### PR DESCRIPTION
**Pull request by GustavoKatel**
Hi this PR adds the recipe for libmodsecurity on linux

### CODE REVIEW

**Comment by GustavoKatel** [comment]
Hi @conda-forge/staged-recipes any update on this?

File: recipes/modsecurity/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  description: |
+    modsecurity is one component of the ModSecurity v3 project. The library codebase serves as an interface to ModSecurity Connectors taking in web traffic and applying traditional ModSecurity processing. In general, it provides the capability to load/interpret rules written in the ModSecurity SecRules format and apply them to HTTP content provided by your application via Connectors.
+  doc_url: https://github.com/SpiderLabs/ModSecurity/tree/v3/master/doc
+  dev_url: https://github.com/SpiderLabs/ModSecurity
+
+extra:
+  recipe-maintainers:
+    - GustavoKatel
+    - Davysson0
```
Comment (mariusvniekerk) [inline recipes/modsecurity/meta.yaml]: @Davysson0 are you okay with being a maintainer on this recipe?

File: recipes/modsecurity/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{% set name = "modsecurity" %}
+{% set version = "3.0.2" %}
+{% set sha256 = "e9d7f4f33ddfc3335073f8c5efa7b6d83bd532535b6b836a300c60fc843d780b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SpiderLabs/ModSecurity/releases/download/v{{ version }}/modsecurity-v{{ version }}.tar.gz
```
[the remark this answers is not in the document]
Comment (GustavoKatel) [reply recipes/modsecurity/meta.yaml]: Isn't it the same url? Unfortunately their build system requires some git information during setup (such as commit ref and so) and I am avoiding this with the patch.

File: recipes/modsecurity/meta.yaml (hunk 1 above)
Comment (Davysson0) [reply recipes/modsecurity/meta.yaml]: Sure

**Pull request merged**